src/main/java/in/ashwanthkumar/gocd/github/util/URLUtils.java

```diff
@@ -0,0 +1,18 @@
+package in.ashwanthkumar.gocd.github.util;
+
+import java.net.URL;
+import java.net.URLConnection;
+
+public class URLUtils {
+    public boolean isValidURL(String url) {
+        boolean isValid = true;
+        try {
+            URL urlObj = new URL(url);
+            URLConnection conn = urlObj.openConnection();
+            conn.connect();
```

Comment:
isn't required i suppose - http://stackoverflow.com/questions/272910/in-java-when-does-a-url-connection-close